src/bin/cargo/commands/clean.rs

```diff
@@ -18,6 +19,11 @@ pub fn cli() -> Command {
             "Package to clean artifacts for",
             ArgValueCandidates::new(get_pkg_name_candidates),
         )
+        .arg(
+            flag("workspace", "Only clean the workspace packages")
+                .short('w')
```

Comment:
We do not generally have s short flag for `--workspace`.  While `cargo update` has one, that is an oversight.